src/components/SendNativeToken/sendNativeTokenUI.tsx

```diff
@@ -0,0 +1,163 @@
+import React, { useState, KeyboardEvent, useEffect } from 'react';
+import {
+  EtherspotBatches,
+  EtherspotBatch,
+  EtherspotTransaction,
+  useEtherspotTransactions,
+} from "@etherspot/transaction-kit";
+import { ethers } from "ethers";
+import { isValidEthereumAddress } from '../../utils/validation';
+
+
+interface SendNativeTokenProps {
+  receiverAddress: string;
+  chain: number;
+  className?: string;
+  style?: React.CSSProperties;
+  unstyled?: boolean;
+  debug?: boolean;
+  triggerElement?: React.ReactNode | React.RefObject<HTMLElement>;
+  disableSendOnEnter?: boolean;
+  onlyEstimate?: boolean;
+}
+
+const SendNativeTokenUI = ({
+  receiverAddress,
+  chain,
+  className,
+  style,
+  unstyled = false,
+  debug = false,
+  triggerElement,
+  disableSendOnEnter = false,
+  onlyEstimate = false,
+}: SendNativeTokenProps) => {
+
+  const [amount, setAmount] = useState('0.001');
+  const [isValidAddress, setIsValidAddress] = useState(true);
+  const [error, setError] = useState('');
+  const { estimate, send } = useEtherspotTransactions();
+  const randomWallet = ethers.Wallet.createRandom();
+  const providerWallet = new ethers.Wallet(randomWallet.privateKey);
+  console.log(providerWallet);
```

Comment:
Random wallet is here again - please remove this and accept a provider that is passed via props from the developer